### PR TITLE
feature:docker image supports JVM parameter injection

### LINKED ISSUE #5521: docker image supports JVM parameter injection
## Why you need it?
 Is your feature request related to a problem? Please describe in details

docker image supports JVM parameter injection

1. Keep the default JVM parameters consistent with the script https://github.com/seata/seata/blob/2.x/distribution/bin/seata-server.sh.
2. Support environment variable injection of JVM parameters such as Xmx, Xms, etc.

docker hub: https://hub.docker.com/repository/docker/seataio/seata-server/general

jib plugin: https://github.com/seata/seata/blob/9b35ba9f6b866f9aaa1ff6e655f6b0e576faca4a/server/pom.xml#L287

---

1. 保持默认JVM参数与脚本https://github.com/seata/seata/blob/2.x/distribution/bin/seata-server.sh一致。

2. 支持JVM参数如Xmx、Xms等的环境变量注入。

docker hub: https://hub.docker.com/repository/docker/seataio/seata-server/general

jib plugin: https://github.com/seata/seata/blob/9b35ba9f6b866f9aaa1ff6e655f6b0e576faca4a/server/pom.xml#L287

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

### PR DESCRIPTION
**Pull request by xingfudeshi**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
1.Unified JVM parameters, they can be viewed in   [seata-setup.sh](https://github.com/seata/seata/blob/1c54323711e9131b5201bb74cdf6328baf7ba6c2/distribution/bin/seata-setup.sh)
2.Docker image supports JVM parameter injection.

### Ⅱ. Does this pull request fix one issue?

fixes #5521 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
I've tested it localy.
1.Set some JVM parameters
image
image
image
2.See results
image

### Ⅴ. Special notes for reviews
1.Use openjdk:8-jre-slim instead of openjdk:8u212-jre-alpine for JAVA8 platform, because the previous base image does not support bash commands.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ install: true
 
 before_script:
   - if [ "$TRAVIS_JDK_VERSION" == "openjdk8" ]; then
-      export IMAGE_NAME="openjdk:8u212-jre-alpine";
+      export IMAGE_NAME="openjdk:8-jre-slim";
```
Comment (slievrly) [inline .travis.yml:16]: This file is currently deprecated.
Note that our base image uses openjdk:8u342

https://github.com/seata/seata/blob/7e40e05c1ba4152798aadf1530d3cd6ee9deaa9e/.github/workflows/publish-docker.yml#L44

Comment (xingfudeshi) [reply .travis.yml:16]: ok.i will check it out.

**Review by slievrly — APPROVED** [review]
After the merger will be validity test workflow.

**Comment by slievrly** [comment]
docker run --name seata-server -p 8091:8091 -p 7091:7091 -e JVM_XMX=1024m -e JVM_XMS=1024m  -d seataio/seata-server:latest
image

**Review by slievrly — COMMENTED** [review]
image
docker run --name seata-server -p 8091:8091 -p 7091:7091 -e JVM_XMX=1024m -e JVM_XMS=1024m  -d seataio/seata-server:latest

**Comment by a364176773** [comment]
同步一份到develop分支提交pr

**Pull request merged**